src/plone/restapi/types/utils.py

```diff
@@ -76,13 +76,20 @@ def get_ordered_fields(fti):
     return (fields, ordered_fieldsets_fields)
 
 
+def get_hidden_fields_from_schema(schema):
```

Comment:
@tisto does this respect schema interface inheritance? (Think schema inheritance chain where a field is omitted at some point, but later down the chain gets un-omitted by yet another directive).

It might be appropriate to use [`mergedTaggedValuesForIRO()` or `mergedTaggedValuesForForm()` from `plone.autoform.utils`](https://github.com/plone/plone.autoform/blob/master/plone/autoform/utils.py#L40-L67) here - these should respect IRO (Interface Resolution Order) and merge tagged values accordingly.